components/ledger/internal/api/v1/controllers_accounts.go

```diff
@@ -7,19 +7,17 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/formancehq/ledger/internal/api/shared"
+	"github.com/formancehq/ledger/internal/api/backend"
```

Comment:
I have a global question here: should we add a lint in order to group imports ? (just my own opinion on it, feel free to tell me what you think)

Reply:
We already use the goimports linter on the main .golangci.yml